src/ViewModel/Converter/Block/FigureAssetVideoConverter.php

```diff
@@ -36,11 +36,16 @@ public function convert($object, string $viewModel = null, array $context = [])
         /** @var Block\Video $asset */
         $asset = $object->getAsset();
 
+        $width = 632;
+        if ($width > $asset->getPlaceholder()->getWidth()) {
+            $width = $asset->getPlaceholder()->getWidth();
+        }
+
         $assetViewModel = new ViewModel\Video(
             array_map(function (Block\VideoSource $source) {
                 return new ViewModel\MediaSource($source->getUri(), new ViewModel\MediaType($source->getMediaType()));
             }, $asset->getSources()),
-            $asset->getPlaceholder() ? $this->iiifUri($asset->getPlaceholder(), $asset->getWidth(), $asset->getHeight()) : null,
+            $asset->getPlaceholder() ? $this->iiifUri($asset->getPlaceholder(), $width) : null,
```

Comment:
Seems like Chrome (at least) still downloads the placeholder even though it autoplays the video. Anything we can do about that @davidcmoulton?

Reply:
@thewilkybarkid I don't think so.